daft/execution/actor_pool_udf.py

```diff
@@ -18,11 +18,19 @@
 
 _SENTINEL = ("__EXIT__", 0)
 
+# Disable python's resource tracking of shared memory objects. Instead, we rely on our own cleanup.
+# The multiprocessing library's resource tracker is does not properly track shared memory objects when the objects are created and
+# unlinked in different processes, although it is allowed to be used in this way.
+# For reference, see these issues:
+# https://github.com/vllm-project/vllm/issues/5468
+# https://stackoverflow.com/questions/62748654/python-3-8-shared-memory-resource-tracker-producing-unexpected-warnings-at-appli
+resource_tracker.unregister = lambda *args, **kwargs: None
+resource_tracker.register = lambda *args, **kwargs: None
```

Comment:
Actually I thought a little bit more about this. I'm a little spooked because we might cause weird issues with other libraries that might use multiprocessing.resource_tracker without our knowledge.

Maybe we keep the `resource_tracker.unregister(shm.name, "shared_memory")`

and here we do something like

```
def safe_unregister(name, rtype):
    try:
        resource_tracker.unregister(name, rtype)
    except KeyError:
        # Ignore if already cleaned up.
        pass

resource_tracker.unregister = safe_unregister
```